constants_test.py

```diff
@@ -122,3 +122,18 @@ def test_language_constants_are_in_sync(self):
             set(rtl_audio_languages) & set(ltr_content_languages)
         )
         self.assertFalse(conflicts)
+
+    def file_path_creation(self):
+        """Tests if creating correct file path."""
+        filename = 'img_20200521_033202_4r4j4ljrwy_height_324_width_432.svg'
+        actual_filepath = constants.create_filepath_with_prefix(
```

Comment:
Just a point:

`actual_filepath` here will be equal to `thumbnail/img_20200521_033202_4r4j4ljrwy_height_324_width_432.svg`
which is not equal to the `expected_filepath` (in the following lines)

If the scope of this method is only to create a part of the url (`filename_prefix/filename`) -- which I think is the case, then update `expected_filepath` to match these values.
(or)
If we want to use filepath to construct complete URL - we should be using fs_domain class. Like how we are implementing in `update_thumbnail_filename()` in story_domain.py

Reply:
Ah, I missed updating the actual filepath. 
Ok, wdyt with which option we should go, complete url is also sounds good to me.

Reply:
Yes, I think having a function to return the fs_domain object also sounds good, but I think that change will have to be made in fs_services file (not sure). This will be useful because, we will need these for thumbnails and rich text images too.

@seanlip WDYT ?

Reply:
I guess it's a bit hard to generalize the `fs_domain` and `fs_service` work because every place has there own requirement.